packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -147,30 +147,52 @@ class TerraService {
    */
   async onComplete(_, config) {
     try {
+      const packageJson = fs.readJsonSync(path.join(process.cwd(), 'package.json'));
+      const repoUrl = new URL(packageJson.repository.url);
+      const repoName = repoUrl.pathname.match(/[^/]+/g);
+      const octokit = createOctokit(this.serviceOptions.gitApiUrl, this.serviceOptions.gitToken);
```

Comment:
I just realized this happens before the check on line 159 below. I will fix that.

Reply:
Addressed as of eb70843f945d3e964464c5f483c67b051faa6a19.